CONTRIBUTING.md

```diff
@@ -1,5 +1,27 @@
+# Contributing to Eclipse Open VSX Publish Extensions
+
 Thanks for stopping by and willing to contribute to `publish-extensions`! Below are some general rules to abide by when contributing to the repository.
 
+## Code of Conduct
+
+This project is governed by the [Eclipse Community Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+
+## Communication
+
+The following communication channels are available:
+
+ * [publish-extensions issues](https://github.com/open-vsx/publish-extensions/issues) - for general issues (bug reports, feature requests, etc.)
+ * [open-vsx.org issues](https://github.com/EclipseFdn/open-vsx.org/issues) - for issues related to [open-vsx.org](https://open-vsx.org/) (outage reports, requests about extensions and namespaces, etc.)
+ * [Developer mailing list](https://accounts.eclipse.org/mailing-list/openvsx-dev) - for organizational issues (e.g. elections of new committers)
+
+## How to Contribute
+
+Before your pull request can be accepted, you must electronically sign the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php).
```

Comment:
Do we have a bot of sorts which we could install here to help with enforcing this rule? I am not sure how I would be able to check incoming Pull Requests 🙏

Reply:
@filiptronicek Good question, I've asked Wayne Beaton to weigh in on that.

Reply:
@filiptronicek Eclipse Foundation has a routine that checks for a signed ECA that gets wired into the PR merge process. This is done for orgs and repos that EF is responsible for. For some reason though, when this deployment moved to EF, this `publish-extensions` repo did not move under EF responsibility. Once we make that move, the ECA check will be handled for committers.